spec/api/v4/imports/imports_spec.rb

```diff
@@ -3,25 +3,34 @@
 describe "Import v4 API", swagger_doc: "v4/import.json" do
   before { Flipper.enable(:imports_api) }
   path "/import" do
+    let(:organization) { FactoryBot.create(:organization) }
+    let(:resource) { build_patient_import_resource }
+    let(:machine_user) { FactoryBot.create(:machine_user, organization: organization) }
+    let(:application) { FactoryBot.create(:oauth_application, owner: machine_user) }
+    let(:token) {
+      FactoryBot.create(:oauth_access_token,
+        application: application,
+        scopes: "write",
+        resource_owner_id: machine_user.id)
+    }
     put "Send bulk resources to Simple" do
       tags "import"
       security [access_token: [], import_auth: ["write"]]
-      parameter name: :organization_header,
+      parameter name: :HTTP_X_ORGANIZATION_ID,
```

Comment:
Making this more consistent with the other API header texts in the generated swagger. Unfortunately, due to how rswag works, there's no way to give the correct name for this API header, which is `X-Organization-ID`. The workaround for now is to explicitly call it out in the description below.